src/components/community-link/community-link.js

```diff
@@ -0,0 +1,81 @@
+import '@brightspace-ui/core/components/link/link.js';
+import '@brightspace-ui/core/components/button/button-icon.js';
+import { html, LitElement } from 'lit';
+import { getDocumentLocaleSettings } from '@brightspace-ui/intl/lib/common.js';
+
+class CommunityLink extends LitElement {
+
+	static get properties() {
+		return {
+			text: { type: String },
+			type: { type: String },
+			articleMap: { attribute: 'article-map', type: Object }
+		};
+	}
+
+	constructor() {
+		super();
+		this._localeSettings = getDocumentLocaleSettings();
+		this._handleUpdate = () => this.requestUpdate();
+	}
+
+	get link() {
+		const { language } = getDocumentLocaleSettings();
+		const { articleCode, langCode } = this._getArticleCode(language);
+
+		return `https://community.d2l.com/brightspace${langCode === 'en' ? '' : `-${langCode}`}/kb/articles/${articleCode}`;
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		this._localeSettings.addChangeListener(this._handleUpdate);
+	}
+
+	disconnectedCallback() {
+		super.disconnectedCallback();
+		this._localeSettings.removeChangeListener(this._handleUpdate);
+	}
+
+	render() {
+		return this.type === 'button'
+			? html`<d2l-button-subtle
+				icon="d2l-tier1:help"
+				text=${this.text}
+				@click="${this._handleButtonClick}"
+			>`
```

Comment:
Custom elements always need a closing tag.
```suggestion
			></d2l-button-subtle>`
```